tests/TestDisplayFunction.php

```diff
@@ -0,0 +1,13 @@
+<?php
+require_once '../Book.php';
+require_once '../displayFunction.php';
+
+use PHPUnit\Framework\TestCase;
+
+class TestDisplayFunction extends TestCase
+{
+    public function testCreateString()
+    {
+        $book = new Book('TitleTest', 'AuthorTest', 'GenreTest', 2023);
```

Comment:
You haven't actually added an assertion in here, you should make sure that the createTile function actually returns the correct HTML